lib/logger/lib/logger/watcher_supervisor.ex

```diff
@@ -10,10 +10,13 @@ defmodule Logger.WatcherSupervisor do
   def start_link({m, f, a}) do
     case Supervisor.start_link(__MODULE__, [], name: @name) do
       {:ok, _} = ok ->
-        _ = for {mod, handler, args} <- apply(m, f, a) do
-          {:ok, _} = watch(mod, handler, args)
-        end
+        _ =
```

Comment:
This formatting seems a little odd to me. I don't think there should have been a newline added here since it wouldn't have been over the character count for the line.